src/cbmc/cbmc_parse_options.h

```diff
@@ -87,35 +85,17 @@ class cbmc_parse_optionst:
     const std::string &extra_options);
 
 protected:
+  goto_modelt goto_model;
   ui_message_handlert ui_message_handler;
-  virtual void register_languages();
-
-  virtual void get_command_line_options(optionst &options);
-
-  virtual int do_bmc(
-    bmct &bmc,
-    const goto_functionst &goto_functions);
-
-  virtual int get_goto_program(
-    const optionst &options,
-    expr_listt &bmc_constraints,
-    goto_functionst &goto_functions);
-
-  virtual bool process_goto_program(
-    const optionst &options,
-    goto_functionst &goto_functions);
-
-  bool set_properties(goto_functionst &goto_functions);
 
   void eval_verbosity();
-
-  // get any additional stuff before finalizing the goto program
-  virtual int get_modules(expr_listt &bmc_constraints)
-  {
-    return -1; // continue
-  }
-
+  void register_languages();
+  void get_command_line_options(optionst &options);
```

Comment:
If the parameter name is removed elsewhere, it seems safe to do so here as well.